### ISSUE TITLE
[Bug] Re-fix the time precision warning in csv_space.py, fix new(?) problem with ps plots

### ISSUE THREAD

**Issue by EdwardSafford-NOAA** [danholdaway/eva#201]
## Current behavior (describe the bug)

I had fixed the time precision warning in `csv_space.py` with #199 but that change is not in develop.  Re-fix that.

I believe there are some other changes I'd made about the same time that are not in develop.  I had stacked up 5 small PRs a couple of weeks ago as we were waiting for the CI test fix so I think some changes got lost on the way.  As I find missing thing I'll add them to this bug report.

**Comment by EdwardSafford-NOAA** [comment]
A new problem.  When plotting ps obs data this error is now generated.  @kevindougherty-noaa does this look familiar at all?  
I've tried the pip install for emcpy with and without the `--no-dependencies` argument but I get this error in both.
```
Traceback (most recent call last):
0:   File "/scratch1/NCEPDEV/da/Edward.Safford/noscrub/git/obs-monitor/ush/plotObsMon.py", line 250, in <module>
0:     eva(plot_yaml)
0:   File "/scratch1/NCEPDEV/da/Edward.Safford/noscrub/eva/opt/eva/eva_driver.py", line 235, in eva
0:     figure_driver(eva_dict, data_collections, timing, logger)
0:   File "/scratch1/NCEPDEV/da/Edward.Safford/noscrub/eva/opt/eva/plotting/batch/base/plot_tools/figure_driver.py", line 145, in figure_driver
0:     make_figure(handler, figure_conf_fill, plots_conf_fill,
0:   File "/scratch1/NCEPDEV/da/Edward.Safford/noscrub/eva/opt/eva/plotting/batch/base/plot_tools/figure_driver.py", line 235, in make_figure
0:     fig.create_figure()
0:   File "/scratch1/NCEPDEV/da/Edward.Safford/noscrub/eva/opt/emcpy/plots/create_plots.py", line 316, in create_figure
0:     plot_dict[layer.plottype](layer, ax)
0:   File "/scratch1/NCEPDEV/da/Edward.Safford/noscrub/eva/opt/emcpy/plots/create_plots.py", line 668, in _horizontalline
0:     ax.axhline(plotobj.y, **inputs)
0:   File "/scratch1/NCEPDEV/da/python/envs/eva/lib/python3.10/site-packages/matplotlib/axes/_axes.py", line 787, in axhline
0:     l = mlines.Line2D([xmin, xmax], [y, y], transform=trans, **kwargs)
0:   File "/scratch1/NCEPDEV/da/python/envs/eva/lib/python3.10/site-packages/matplotlib/lines.py", line 407, in __init__
0:     self._internal_update(kwargs)
0:   File "/scratch1/NCEPDEV/da/python/envs/eva/lib/python3.10/site-packages/matplotlib/artist.py", line 1219, in _internal_update
0:     return self._update_props(
0:   File "/scratch1/NCEPDEV/da/python/envs/eva/lib/python3.10/site-packages/matplotlib/artist.py", line 1193, in _update_props
0:     raise AttributeError(
0: AttributeError: Line2D.set() got an unexpected keyword argument 'type'
srun: error: h21c52: task 0: Exited with exit code 1
```

**Comment by kevindougherty-noaa** [comment]
@EdwardSafford-NOAA hmm I have not seen this before. That is another interesting one.

**Comment by EdwardSafford-NOAA** [comment]
@kevindougherty-noaa Ok, I'll dig a bit deeper and let you know what I learn.

**Comment by EdwardSafford-NOAA** [comment]
No solution yet.  I'm working this on hera were all plots were working as recently as 2 weeks ago.   So far I've tried these things:

- confirmed emcpy, wxflow, eva, and obs-monitor are the latest versions.
- reinstalled everything without the --no-dependencies flag (which solves some things on orion|hercules)
- noting that matplotlib was installing version 3.9.1 while eva requires 3.9.0, I did a --force-reinstall "matplotlib==3.9.0" to get the older version
- confirmed that `emcpy_horizontal_line.py` is removing `type` from the figure config dictionary

Still no idea where/how `type` is getting to matplotlib or why it's suddenly a problem.
